test/EasyCaching.UnitTests/TestHelpers/TestsRedisConstants.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace EasyCaching.UnitTests
+{
+    public static class TestsRedisConstants
+    {
+        private const string RedisHost = "redis";
+        public const string AvailableRedis = RedisHost + ":6379,allowAdmin=true";
+        public const string AvailableRedisWithDisallowedAdmin = RedisHost + ":6379,allowAdmin=false";
+        public const string UnavailableRedis = RedisHost + ":9999,abortConnect=true,connectTimeout=1";
+        public const string UnavailableRedisWithAbortConnect = RedisHost + ":9999,abortConnect=false,connectTimeout=1";
+
+        public static string GetAvailableRedisConnectionString(int? defaultDatabase = null) =>
```

Comment:
```suggestion
        public static string GetAvailableRedisWithDatabase(int defaultDatabase) =>
```
Так понятнее, чем отличается AvailableRedis от этого метода